### DISCUSSION TITLE
Absolute Paths / Aliases in

### DISCUSSION THREAD

**Discussion by developersaintt** [saltyshiomix/nextron#294] [Q&A]
Unable to add path aliases. I have also tried [Module Alias](https://www.npmjs.com/package/module-alias) but it didn't work also. Kindly help.

`
  "_moduleAliases": {
    "@/prisma/*": ["prisma/*"],
    "@/components/*": ["src/components/*"],
    "@/config/*": ["src/config/*"],
    "@/hooks/*": ["src/hooks/*"],
    "@/layouts/*": ["src/layouts/*"],
    "@/lib/*": ["src/lib/*"], 
    "@/providers/*": ["src/providers/*"],
    "@/sections/*": ["src/sections/*"],
    "@/styles/*": ["src/styles/*"]
  },
`

also tried putting the components in the renderer folder too. I keep getting this error 
![image](https://user-images.githubusercontent.com/31191699/200822210-27792625-e6ca-4c36-8d92-a287479177c7.png)

**Comment by Willem-Jaap** [comment]
This should be fixed if you follow the answer provided in this issue: [Issue #235](https://github.com/saltyshiomix/nextron/issues/235)
